### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Stephen-Adom**
# Components
In this pull request, the following tasks were implemented:
- Created a new directory called components.
- Used functional components
- Created at least 1 child component

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @Stephen-Adom 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has empty section. Kindly customized it to make your README more professional 👍 
Screenshot 2023-03-20 at 8 56 12 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Math magicians is a website for all fans of mathematics
 
 ### Key Features <a name="key-features"></a>
 
+- **Calculator**
+
```
Comment (zieeco) [inline README.md:49]: ### FUTURE FEATURES


- You have submitted a project with an empty "FUTURE FEATURES" section. Kindly add some future features about your project to make your README more professional +1 

Screenshot 2023-03-20 at 8 56 12 PM


- All your `back-to-top` links in the readme is not working, ensure you fix this issue so they link to the corresponding top navigation section of the readme file.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Stephen-Adom ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -37,12 +37,15 @@ Math magicians is a website for all fans of mathematics
 ### Tech Stack <a name="tech-stack"></a>
 
 - **ReactJS**
+- **Bootstrap**
 - **CSS 3**
 
 <!-- Features -->
```
Comment (thecodechaser) [inline README.md:45]: - [ ] As requested by the previous reviewer, All your `back-to-top` links in the README file are not working, Please make sure to fix this issue so they link to the corresponding top navigation section of the README file.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Stephen-Adom👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
## Hi @Stephen-Adom 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 

giphy (2)

## Highlights

✅ Good commit history
✅ Linters are passing

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**